### DISCUSSION TITLE
Request for Echidna benchmark contracts

### DISCUSSION THREAD

**Discussion by ddrenteria** [crytic/echidna#1054] [General]
We are working for our thesis on some changes to the fuzzing feedback method in Echidna. We would like to be able to access the contracts you used for your comparisons in your paper (_Echidna: Effective, Usable, and Fast Fuzzing for Smart Contracts_) to benchmark our changes (that would save us some trouble).
Thanks in advance!!

**Comment by gustavo-grieco** [comment]
Hi, I'm really sorry for the delay answering this. We don't have access to these contracts, but they shouldn't be too hard to replicate. In any case, for getting a fair and useful benchmark, we strongly recommend NOT to use these, since they are old, outdated and very simplistic code. We have a few recommendations for a corpus, but based on real DeFi contracts, instead of snippets of code. Do you mind to write us to echidna-dev@trailofbits.com or joining a discussion in [EmpireHacking](https://empireslacking.herokuapp.com/) to start this discussion?
